face_rating/processing/face_detector.py

```diff
@@ -1,13 +1,82 @@
 #!/usr/bin/env python3
 
+from typing import List
 import numpy as np
+
 import cv2
 
-face_cascade = cv2.CascadeClassifier('haarcascade_frontalface_default.xml')
+FACE_EXTRA_TOP_PERC = 0.25  # Percentage of extra pixels to include on top of face
+WHITE = (255, 255, 255)
+
+
+def get_resizing_parameters(image_size, target_size):
+    '''
+    Get width, height and top, bottom, left, right border sizes
```

Comment:
No need to have newlines if it fits in one line

"""Hello I fit on one line."""

Also, please add periods at the end of sentences :)